### ISSUE TITLE
Unable to bind picker value

### ISSUE THREAD

**Issue by Mero-Plaform** [mskocik/svelty-picker#110]
I'm using:
"svelty-picker": "^4.0.3"
"svelte": "^3.57.0"

----

The ```bind:value``` directive suppose to make to way binding

but it works only as initial value at first render and receving values from picker

in example below if ```variable``` value is changing it doesn't affect to ```SveltyPicker```

so I'm just unable to change picker value if it's needed without user manual time choose

```
<script>
  import SveltyPicker from "svelty-picker";
  
  let variable = "01:00";
</script>


<SveltyPicker
    bind:value={variable}
    format="hh:ii"
 />
{variable}
<input bind:value={variable} />
```

**Comment by MihailZharavin11** [comment]
I upgraded sveltypicker to version 4.0.5. I'm having the same issue. 
<SveltyPicker bind:value={$SomethingState.date}/>.
Inside some function I set $SomethingState.date = null; 
But sveltyPicker still continues to show the past date and does not reset it

**Comment by mskocik** [comment]
It works for me? [Repl here](https://svelte.dev/repl/e75bae5f495349b3a9ad987ccfbbddde?version=4.1.1)

**Comment by MihailZharavin11** [comment]
But if I set null in the function, sveltypicker still displays today's date, but I want the field to be empty

**Comment by mskocik** [comment]
Issue was only with 'display date'. Value was already reset correctly. Display date fixed in 4.0.6
